static/templates/settings/deactivate-realm-modal.handlebars

```diff
@@ -0,0 +1,13 @@
+<div id="deactivate-realm-modal" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivate_realm_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="deactivate_realm_modal_label">{{t "Deactivate organization" }} <span class="realm_name"></span></h3>
+    </div>
+    <div class="modal-body">
+        <p>{{t "This action is permanent and cannot be undone. All users will permanently lose access to their Zulip accounts." }}</p>
```

Comment:
@rishig I think maybe "and can only be undone by a server administrator".

Reply:
I think we don't want support requests for this, though? Also does deactivating + reactivating have no side effects?

Reply:
If we do want to give that possibility, I could imagine "to reactivate, you'll need to contact ZULIP_ADMINISTRATOR", so we don't have to explain what a server admin is.

Reply:
It has basically no side effects other than logging everyone out.  So, in practice, it's easy to reactivate (there's a management command).

But I guess people doing it by accident is unlikely and they'll contact support anyway after seeing the page for a deactivated realm, so fine, we can leave this as-is.